app/wizards/placements/add_school_contact_wizard/school_contact_step.rb

```diff
@@ -3,13 +3,14 @@ class Placements::AddSchoolContactWizard::SchoolContactStep < Placements::BaseSt
   attribute :last_name
   attribute :email_address
 
-  validates :first_name, :last_name, presence: true
```

Comment:
Why are we removing this validation rule? Was it not supposed to be here in the first place?

Reply:
There is no validation on the SchoolContact model for these attributes. This is to allow for shared inboxes.